src/IBM.Watson.VisualRecognition.v3/VisualRecognitionService.cs

```diff
@@ -28,23 +32,27 @@
 
 namespace IBM.Watson.VisualRecognition.v3
 {
+    [System.Obsolete("On 1 December 2021, Visual Recognition will no longer be available. " +
+        "For more information, see Visual Recognition Deprecation " +
+        "(https://github.com/watson-developer-cloud/java-sdk/tree/master#visual-recognition-deprecation).")]
```

Comment:
link is pointing to `java-sdk`

Reply:
good catch!

Reply:
fixed!

Reply:
no worries, I almost did the same for unity at first 😂